go/Makefile

```diff
@@ -15,4 +18,24 @@ clean:
 	rm -f pkg/sdk/optimism/abis/*.go
 	rm -f pkg/sdk/ethereum/abis/*.go
 
-test :; go test github.com/NovaSubDAO/nova-sdk/go/...
+test: build
```

Comment:
can we put the script below in a file `run_tests.sh` and then we simply replace this line with `test: run_tests.sh`

Reply:
Done